### PR TITLE
orchestrator: a new interface for enabling/disabling orchestration te…

### PR DESCRIPTION
**Pull request by mitake**
…mporally

/cc @AkihiroSuda I addressed your comments on my local fork

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
CI failing

File: nmz/signal/interface.go [hunk 1]
```diff
@@ -60,6 +60,15 @@ type Action interface {
 	Event() Event
 }
 
+const (
+	ControlEnableOrchestrator = iota
```
Comment (mitake) [inline nmz/signal/interface.go]: I think both of them are ok

**Comment by mitake** [comment]
@AkihiroSuda fixed the CI failure, PTAL

File: nmz/signal/interface.go (hunk 1 above)
Comment (AkihiroSuda) [inline nmz/signal/interface.go]: question: why REST API uses "orchestration"?

**Comment by AkihiroSuda** [comment]
lgtm

**Pull request merged**